wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientTableViewCell.swift

```diff
@@ -119,79 +93,59 @@ class ClientTableViewCell: UITableViewCell, DynamicTypeCapable {
         let textColor = SemanticColors.Label.textDefault
         nameLabel.accessibilityIdentifier = "device name"
         labelLabel.accessibilityIdentifier = "device label"
-        activationLabel.accessibilityIdentifier = "device activation date"
-        fingerprintLabel.accessibilityIdentifier = "device fingerprint"
-        verifiedLabel.accessibilityIdentifier = "device verification status"
-
-        activationLabel.numberOfLines = 0
-        activationLabel.textColor = textColor
+        proteusIdLabel.accessibilityIdentifier = "device fingerprint"
 
-        fingerprintLabelFont = .smallLightFont
-        fingerprintLabelBoldFont = .smallSemiboldFont
-        fingerprintTextColor = textColor
+        proteusIdLabel.accessibilityIdentifier = "device proteus ID"
+        mlsThumbprintLabel.accessibilityIdentifier = "device mls thumbprint"
+        mlsThumbprintLabel.numberOfLines = 1
+        proteusIdLabel.numberOfLines = 1
```

Comment:
question: Are we blocking dynamic type from working by setting the numberOfLines = 1?

Reply:
These details get truncated. So this should be fine